src/metrics/filters/tags_filter.go

```diff
@@ -120,9 +120,6 @@ type TagsFilterOptions struct {
 
 	// Function to extract name and tags from an id.
 	NameAndTagsFn id.NameAndTagsFn
-
-	// Function to create a new sorted tag iterator from id tags.
-	SortedTagIteratorFn id.SortedTagIteratorFn
```

Comment:
do you no longer need this because the MatchOpts are passed to the TagFilter?

Reply:
Exactly yeah. I think this was already dead code by the time I got here (they were already passed to the TagFilter on the callpath): https://github.com/m3db/m3/blob/4445aa308190c85b517608e80e2c6ce9c82faf91/src/metrics/filters/tags_filter.go#L186